src/emails/templates/org-invite.tsx

```
@@ -0,0 +1,154 @@
+  // Choose the appropriate message based on theme name
+  if (themeName === "ol-learn") {
+    return (
+      <EmailLayout preview={`Invitation to join MIT Learn`} locale={locale}>
+        <Text style={paragraph}>
+          <p>
+            Hi,
+          </p>
+
+          <p>
+            You have been invited to join <span style={emphasizedText}>MIT Learn</span>. 
+            This platform provides access to MIT Learn's educational resources and course materials.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)